app/proto/account.proto

```diff
@@ -93,17 +94,10 @@ message ChangeEmailReq {
 
 message GetContributorFormInfoRes {
   bool filled_contributor_form = 1;
-
-  string username = 2;
-  string name = 3;
-  string email = 4;
-  uint32 age = 5;
-  string gender = 6;
-  string location = 7;
 }
 
-message MarkContributorFormFilledReq {
-  bool filled_contributor_form = 1;
+message FillContributorFormReq {
+  org.couchers.auth.ContributorForm contributor_form = 1;
```

Comment:
It this extra indirection to be backwards compatible for future expansions?

Reply:
Ah, yes; I was going to mention that. I've added it into the list of things to think about for API refactoring: #899, but decided to do it this way for now.